src/graphql/admin/schema.graphql

```diff
@@ -25,6 +25,8 @@ enum AccountLevel {
 enum AccountStatus {
   ACTIVE
   LOCKED
+  NEW
+  PENDING
```

Comment:
is pending needed? is there a difference between `new` and `pending`?

in both case, if the account needs manual approval to get active, the action will be limited on the account between those 2 status, so maybe we don't need to separate them?

Reply:
New: Account is fresh (no status verifications happened yet). The UI sends the user to the verification flow
Pending: A verification request has been made but the result is not back yet. The UI can either block the user or limit what they can do.